storage/ndb/rest-server/rest-api-server/resources/config/config_localhost.json

```diff
@@ -2,14 +2,6 @@
         "Internal": {
                 "APIVersion": "0.1.0"
         },
-        "REST": {
-                "ServerIP": "localhost",
-                "ServerPort": 4406
-        },
-        "GRPC": {
-                "ServerIP": "localhost",
-                "ServerPort": 5406
-        },
```

Comment:
Did you test this? Are you sure the default ServerIP/Port is not going to be overwritten by "" if you do this?

Reply:
Not sure what you mean, I have tested it and all the tests pass